src/main/web_src/src/app/App.js

```diff
@@ -51,7 +50,9 @@ export default class App extends Component {
 		if (this.state.error)
 			return (
 				<ErrorBoundary
-					error={'Problemer med å hente dolly config. Prøv å refresh siden (ctrl + R).'}
+					error={
+						'Problemer med å hente dolly config eller gyldige miljøer. Prøv å refresh siden (ctrl + R).'
```

Comment:
Kunne kanskje kostet på oss en egen state for config og en for miljøer? Så kan det lages enda mer presise feilmeldinger? 😁

Reply:
Hehe det er helt sant, skal fikse 😄